services/tinlake/actions.ts

```diff
@@ -121,21 +124,19 @@ export const issue = async (
     // approve proxy to take nft if not yet happened
     if (!(await tinlake.isNFTApprovedForAll(nftRegistryAddress, user, proxyAddress))) {
       try {
-        await tinlake.setNFTApprovalForAll(nftRegistryAddress, proxyAddress, true)
+        const approveTx = await tinlake.setNFTApprovalForAll(nftRegistryAddress, proxyAddress, true)
+        await tinlake.getTransactionReceipt(approveTx)
       } catch (e) {
         return loggedError(e, 'Could not approve proxy to take NFT.', tokenId)
       }
     }
 
     // transfer issue
-    let result
     try {
-      result = await tinlake.proxyTransferIssue(proxyAddress, nftRegistryAddress, tokenId)
+      return await tinlake.proxyTransferIssue(proxyAddress, nftRegistryAddress, tokenId)
     } catch (e) {
       return loggedError(e, 'Could not Issue loan.', tokenId)
     }
```

Comment:
We can drop this here as well, right?

Reply:
Yes, but we made those changes in the V3 PR. I don't think we need to duplicate those changes in this PR as well, right? We can just merge those together.